src/modules/homepage/IncomeExpense.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Stack } from '@mui/material';
+import { ReactNode } from 'react';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+import CameraAltIcon from '@mui/icons-material/CameraAlt';
+import { Regular3 } from '../../shared-components/Typography/Regular3';
+
+interface IncomeExpenseCardProps {
+  children: ReactNode;
+  bgcolor: string;
+  color: string;
+  content: string;
+  amount: string;
+}
+
+export const IncomeExpenseCard: React.FC<IncomeExpenseCardProps> = ({
+  children,
+  bgcolor,
+  color,
+  content,
+  amount,
+}) => {
+  return (
+    <Box
+      borderRadius={theme.shape.borderRadius * 7}
+      bgcolor={bgcolor}
+      display='inline-flex'
+      flexDirection='row'
+      alignItems='center'
+      gap='10px'
```

Comment:
All the values like this should be used from `theme.spacing()`, don't hardcode it in px. Applies everywhere in this component.

Reply:
fixed